contracts/upgradeable_contracts/BasicHomeBridge.sol

```diff
@@ -74,9 +77,24 @@ contract BasicHomeBridge is EternalStorage, Validatable {
         if (signed >= reqSigs) {
             setNumMessagesSigned(hashMsg, markAsProcessed(signed));
             emit CollectedSignatures(msg.sender, hashMsg, reqSigs);
+
+            address feeManager = feeManagerContract();
```

Comment:
consider to add the call of `onSignaturesCollected` here and move these four lines to the implementation of this method that will be located in the `HomeBridgeErcToNative` contract. It will be similar we have for the `onExecuteAffirmation` method.

Reply:
Thanks, added `onSignaturesCollected` https://github.com/poanetwork/poa-bridge-contracts/pull/132/commits/1993efcf4c6831fd5cafbfe15c8788a426da9d6b